eframe/src/lib.rs

```diff
@@ -95,9 +100,44 @@ pub use web_sys;
 ///     eframe::start_web(canvas_id, Box::new(|cc| Box::new(MyEguiApp::new(cc))))
 /// }
 /// ```
+#[cfg(target_arch = "wasm32")]
+use arc_swap::ArcSwapOption;
+
+#[cfg(target_arch = "wasm32")]
+unsafe impl Send for AppRunner {}
+
+#[cfg(target_arch = "wasm32")]
+unsafe impl Sync for AppRunner {}
+
+#[cfg(target_arch = "wasm32")]
+static GLOBAL_HANDLE: ArcSwapOption<Mutex<AppRunner>> = ArcSwapOption::const_empty();
+
 #[cfg(target_arch = "wasm32")]
 pub fn start_web(canvas_id: &str, app_creator: AppCreator) -> Result<(), wasm_bindgen::JsValue> {
-    web::start(canvas_id, app_creator)?;
+    let handle = web::start(canvas_id, app_creator)?;
+
+    GLOBAL_HANDLE.swap(Some(handle));
```

Comment:
what should be the semantics of calling `start` many times? Should we warn about it? Destroy the previous instances first?